ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -163,6 +288,13 @@
   @Default("false")
   public final boolean cosmosDirectHttps;
 
+  /**
+   * Azure storage account info.
+   */
+  @Config(AZURE_STORAGE_ACCOUNT_INFO)
+  @Default("")
+  public final String getAzureStorageAccountInfo;
```

Comment:
Some of the existing configs, like `azureStorageConnectionString`, `azureStorageScope`, `azureStorageEndpoint` and a few other might conflict with this config (in the sense that it doesn't make sense for both the configs to co-exist). We should do some validations about that in the `AzureCloudConfig` constructor.

Reply:
I think we better keep it for now. Will implement the logic in a way to give precedence to AZURE_STORAGE_ACCOUNT_INFO